ARCL/Source/LocationNode.swift

```diff
@@ -108,7 +113,7 @@ open class LocationAnnotationNode: LocationNode {
     ///   - view: The view to display at the specified location.
     public init(location: CLLocation?, view: UIView) {
         let plane = SCNPlane(width: view.frame.size.width / 100, height: view.frame.size.height / 100)
-        plane.firstMaterial!.diffuse.contents = view
+        plane.firstMaterial!.diffuse.contents = view.layer
```

Comment:
can you explain what this change is about? it's not mentioned in your PR and it's not clear why it's needed.

Reply:
Oh sorry, that's because I forgot to create a new branch on my fork.
Anyway this is a bug fix when you trying to use a view with transparent background.

Reply:
I've deleted the changes